createst.py

```diff
@@ -344,6 +344,7 @@ def parse_args():
                         help="Create filter blocks for the specified events")
     parser.add_argument("--strictcsums", default=None, action="store_true",
                         help="Stricly validate checksum")
+    parser.add_argument("--features", type=str, action="store_true", required=True, help="Include features defined globally")
```

Comment:
Hi @VictoryWekwa !
When I run your code, I get
```
Traceback (most recent call last):
  File "../suricata-verify/createst.py", line 415, in <module>
    main()
  File "../suricata-verify/createst.py", line 408, in main
    init_global_params()
  File "../suricata-verify/createst.py", line 67, in init_global_params
    args = vars(parse_args())
  File "../suricata-verify/createst.py", line 347, in parse_args
    parser.add_argument("--features", type=str, action="store_true", required=True, help="Include features defined globally")
  File "/usr/lib64/python3.8/argparse.py", line 1368, in add_argument
    action = action_class(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'type'
```
Is it perhaps the Python version issue or argparse version issue? Which Python versions are you testing?